srcpkgs/linux-lts/template

```diff
@@ -1,25 +1,15 @@
 # Template file for 'linux-lts'
 pkgname=linux-lts
-version=4.14
-revision=2
+version=5.4
```

Comment:
I would have thought linux5.10 would be the preferred new LTS, since it's expected to be maintained through 2026 while 5.4 will be maintained through 2024.

Reply:
2025, according to https://www.kernel.org/category/releases.html

They are basically one year apart... I'm not sure which is best, tbh.

Reply:
I'm in favor of 5.10, which seems to run quite well.

Reply:
In that case, would a reasonable policy be to update `linux-lts` to the latest LTS, when it appears? So basically update the package once a year.

Reply:
when it appears + some wait time, maybe? not sure

Reply:
I got 3 votes on 5.10, so I have moved to that.